features/202-session-management/api.md

```diff
@@ -0,0 +1,66 @@
+# Session Management APIs
+
+## SDK API
+```typescript
+interface UserAgent {
+    raw: string;
+    name: string;
+    version: string;
+    os: string;
+    osVersion: string;
+    app: string;
```

Comment:
would it be used for browsers when it is from web? Or there should be separate attributes for web browsers?

Reply:
@kiootic I think you resolved this comment accidentally?

Reply:
I had updated the API doc to unify this: the name would mean the browser name or app ID, and version would mean browser version or app version.